mozilla/fx-private-relay
# fix MPP-2308: Reply to last sender

This PR fixes MPP-2308.

How to test:
1. `PHONES_ENABLED=0 pytest` should run with no errors nor failures
2. `PHONES_ENABLED=1 pytest` should run with no errors nor failures
3. Set up [end-to-end local phone dev](https://github.com/mozilla/fx-private-relay/blob/main/docs/end-to-end-local-phone-dev.md)
4. Send your local relay number a text message
   * [x] You should receive the text message to your real phone
6. Reply to the text message sent by relay
   * [x] The original sender should receive the reply from the relay number
7. As the original sender, reply to the reply
   * [x] You should receive the reply to the reply to your real phone
8. Repeat as desired
